pkgs/top-level/python-packages.nix

```diff
@@ -5512,6 +5512,8 @@ in {
     buildInputs = with self; [ fudge_9 nose ];
   };
 
+  faulthandler = if ! isPy3k then callPackage ../development/python-modules/faulthandler {} else throw "faulthandler is built into ${python.executable}";
```

Comment:
This line is very long, I think it would be clearer as
```
faulthandler =
  if ! isPy3k
  then callPackage ../development/python-modules/faulthandler {}
  else throw "faulthandler is built into ${python.executable}";
```